client/sniffer/src/test/resources/es7_nodes_publication_address_format.json

```diff
@@ -0,0 +1,30 @@
+{
+  "_nodes": {
+    "total": 8,
+    "successful": 8,
+    "failed": 0
+  },
+  "cluster_name": "elasticsearch",
+  "nodes": {
+    "ikXK_skVTfWkhONhldnbkw": {
+      "name": "m1",
+      "transport_address": "127.0.0.1:9300",
+      "host": "127.0.0.1",
+      "ip": "127.0.0.1",
+      "version": "6.0.0",
+      "build_hash": "8f0685b",
+      "roles": [
+        "master",
+        "ingest"
+      ],
+      "attributes": { },
+      "http": {
+        "bound_address": [
+            "dummy.com:9200"
```

Comment:
Can we convert this domain to be something that does not resolve, or isn't an unrelated address for a third party? I would prefer if it were something like "elastic.test" which should never resolve because of the .test TLD guarantees.

Reply:
Make sense. I've updated the domain name.